components/brave_wayback_machine/brave_wayback_machine_utils.cc

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_wayback_machine/brave_wayback_machine_utils.h"
+
+#include <string>
+
+#include "net/base/url_util.h"
+#include "url/gurl.h"
+
+bool IsWaybackMachineDisabledFor(const GURL& url) {
+  if (net::IsLocalhost(url))
+    return true;
+
+  if (url.host().find(".local") != std::string::npos)
```

Comment:
Does Chromium have a `StringEndsWith()` function?

It's not a huge deal, but it would avoid matching things like `www.local-news.com`.

Reply:
Done. thanks for checking!